tools/clang/tools/dxcompiler/dxcutil.cpp

```diff
@@ -108,6 +111,9 @@ void GetValidatorVersion(unsigned *pMajor, unsigned *pMinor) {
 void AssembleToContainer(AssembleInputs &inputs) {
   CComPtr<AbstractMemoryStream> pContainerStream;
   IFT(CreateMemoryStream(inputs.pMalloc, &pContainerStream));
+  if (inputs.SerializeFlags & SerializeDxilFlags::SetRootSignature) {
```

Comment:
If removing SetRootSignature flag, this can change to:
```cpp
if (!(inputs.SerializeFlags & SerializeDxilFlags::StripRootSignature) && inputs.pRootSigBlob) {
```